index.js

```diff
@@ -78,15 +104,47 @@ function getServices(offset) {
           bdk.log.warn('service missing name', service);
         }
       } else {
-        bdk.log.warn('broken service', service);
-        bdk.log.warn('full list of services', services);
+        bdk.log.warn('broken service');
```

Comment:
Is this a meaningful log message for an eventual investigation?

Reply:
yes

Reply:
Last time we had a problem in prod, Jonathan was not able to troubleshoot because the message was just "broken service" - but if you think its ok... I am good

Reply:
im pretty sure what was breaking it was that we were trying to print out the broken service/ the list of services